codeclimate-wrapper

```diff
@@ -41,10 +41,20 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
-exec docker run \
+if [ -t 1 ]; then
+  exec docker run \
   --interactive --tty --rm \
```

Comment:
Can you shift these two spaces so the continuations are still indented beyond what they continue?